### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PAVBAN95**

### CODE REVIEW

**Comment by PAVBAN95** [comment]
Project's name changed :-P

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# 4th-year-project
+# snapAll
 
 This repository contains development and deployment files of 4th year Major Project
```
Comment (bimalkant-lauhny) [inline README.md]: Please add a small description of the project, means, what this project is all about.

**Comment by code-master5** [comment]
@PAVBAN95 Please see requested changes.

File: README.md (hunk 1 above)
Comment (PAVBAN95) [reply README.md]: okay!

**Review by bimalkant-lauhny — APPROVED** [review]

**Pull request merged**